### PR TITLE
add gpp, lzfse, update qemu => 9.0.0

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=qemu crew update
```

### CODE REVIEW

File: packages/gpp.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+     armv7l: '250d4545d879fb41bcacd52fbb2ba52b3a5d2ca2b7517f3e539e518a447d3706',
+     i686: '1be2619edcb83802790428e1663cd2317436d667ac5e416ce2401a7f1a6a437f',
+     x86_64: '6b373f11e39a740775caacaf6d7affaadd9327a33d791a636ac5a26b7f38ccea'
+  })
+
+  depends_on 'glibc' # R
+
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    FileUtils.ln_s "#{CREW_PREFIX}/bin/gpp", "#{CREW_DEST_PREFIX}/bin/#{CREW_TGT}-gpp"
```
Comment (uberhacker) [inline packages/gpp.rb:27]: Why is this symlink needed?

File: packages/lzfse.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+require 'buildsystems/cmake'
+
+class Lzfse < CMake
+  description 'The LZFSE compression library and command line tool'
+  homepage 'https://github.com/lzfse/lzfse'
+  version '1.0'
+  license 'BSD'
+  compatibility 'all'
+  source_url 'https://github.com/lzfse/lzfse.git'
+  git_hashtag 'lzfse-1.0'
```
Comment (uberhacker) [inline packages/lzfse.rb:13]: `git_hashtag "lzfse-#{version}"`?

Comment (satmandu) [reply packages/lzfse.rb]: done

File: packages/gpp.rb (hunk 1 above)
Comment (satmandu) [reply packages/gpp.rb]: ccache was complaining.... reverted as not needed.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**